### ISSUE TITLE
[Feature Request]: Update Juggernaut 8 to Juggernaut X

### ISSUE THREAD

**Issue by jaimeastorga2000** [lllyasviel/Fooocus#2897]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do?

Just saw the commit to update animaPencilXL_v100 to animaPencilXL_v310, which is great, but Juggernaut has also released new models. Request a commit updating juggernautXL_v8Rundiffusion to [juggernautXL_v9Rundiffusionphoto2](https://civitai.com/models/133005?modelVersionId=348913) or [juggernaut_X_RunDiffusion](https://civitai.com/models/133005?modelVersionId=456194) (unsure whether X is preferable because it is apparently different enough from 8 and 9 to require [it's own prompting guide](https://storage.googleapis.com/run-diffusion-public-assets/Prompting_Juggernaut_XL_and_Hyper_Guide_v2_By_Adam_Stewarts.pdf)).

Local users can just download the new models themselves, but this will really help Colab users.

### Proposed workflow

1. Go to .... 
2. Press ....
3. ...

**Comment by ILikeToasters** [comment]
Juggernaut X should not be the default. It has much work to be done since they revamped it. It is debatable if 9 is better than 8 in my opinion. As for Colab you can have it download any model you want. Best way is to actually upload a checkpoint into Google Drive and share a link so you can have Colab download it instantly.

**Comment by AgentHitmanFaris** [comment]
Nah that will have so much works...as foocuss focus on simplfy things, using that module need users do couple of tweaks, V9 is good enough for [juggernaut_X_RunDiffusion](https://civitai.com/models/133005?modelVersionId=456194) not really suitable for this but if want to try can change default sampler to euler_a to get quite clean result and dont set cfg to high 1-2 is good enough

**Comment by DavidDragonsage** [comment]
I agree with our esteemed colleague who has a preference for toasters. Juggernaut X should not replace Juggernaut 8.

Juggernaut X should have really been designated as a Beta because it has many issues, one of the most glaring of which is its obsession with creating nudity - often in quite ridiculous ways. The fact that Juggernaut X supposedly needs elaborate prompts to solve the problems inherent in this model is also against what Fooocus stands for, as an easy to use UI that can create beautiful images with just a couple of words.

I really like both Juggernaut 8 and Juggernaut 9 - and I agree that neither one of these two is really better than the other. So when Juggernaut X came out I was quite excited to try it - and I was very impressed by the level of detail, especially hair detail, in closeup portraits.

For a while I went along with the idea that Juggernaut X needed complex prompts to work properly, and so that is what I used. But I came to realize that it ignored most of what I was prompting so that was a wasted effort. For example, although I specified that the female subject was wearing clothing and had heavily weighted negative prompts against NSFW content, the model would insist on generating randomly placed and anatomically unlikely nipples on a woman's breast.

And then there was the tendency for twenty year old subjects to be given neck and radial chest wrinkles that although possible, are fairly unlikely, especially when I had not specified wrinkles in the prompt. This problem was only partly reduced by placing "smooth skin" in the prompt, but the problem was even more annoying and unlikely when all this wrinkling was present on young Afro subjects.

My conclusion is that Juggernaut X is unfortunately not suitable to be the default base model for Fooocus.

**Comment by MohammadAboulEla** [comment]
I can confirm that Juggernaut 9 is better than Juggernaut X, also Juggernaut 8 + loras  may be better than Juggernaut 9 +  loras  but I'm not sure about that. Juggernaut 9 Lighting itself may gives you the best and the fastest results with only 4:6 steps and CFG of 1:2 (ddpm_sde, karras). the problem is you need higher CFG for non realistic styles. and no preset will work for all the styles so you may try to keep the CFG as high as you can (max of 4) and lower the mimc CFG until you get the the best look for selected style.

**Comment by parth9361** [comment]
yeah please add the version 9 as the default one
